code/game/objects/effects/decals/crayon.dm

```diff
@@ -697,54 +866,58 @@ obj/item/scroll/attackby(obj/item/I, mob/living/carbon/human/M)
 			M.maxHealth += 1
 			M.health += 1
 			M.unnatural_mutations.total_instability += 1 //A soft cap
-		B.remove_self(70)
-		lesser.dust()
-		M.sanity.changeLevel(-20)
+			B.remove_self(70)
+			lesser.dust()
+			M.sanity.changeLevel(-10)
 		return
 	return
 
+// Cards: Invokes a random carp card, to use with other spells
 /obj/effect/decal/cleanable/crayon/proc/cards_spell(mob/living/carbon/human/M)
 	var/datum/reagent/organic/blood/B = M.get_blood()
-	to_chat(M, "<span class='warning'>A voice whispers infront you. Any foils?</span>")
+	to_chat(M, "<span class='warning'>A voice whispers in front of you: Any foils?</span>")
 	for(var/obj/item/device/camera_film in oview(1)) // Must be on the spell circle
 		B.remove_self(15)
-		new /obj/random/card_carp(src.loc)
+		new /obj/item/pack_card_carp(src.loc) // For its blood price this feels more fair.
```

Comment:
```suggestion
		new /obj/random/card_carp(src.loc)
```
Buy them at a vender!

Reply:
Can we then nerf the blood cost? 15 blood for 1 card feels a bit too much